manager/src/main/java/io/aklivity/zilla/manager/internal/commands/install/ZpmInstall.java

```diff
@@ -162,21 +166,24 @@ public void invoke()
                 repositories.removeIf(r -> !r.location.startsWith("file:"));
             }
 
-            ZpmCache cache = new ZpmCache(repositories, cacheDir);
+            List<RemoteRepository> remoteRepositories = getRemoteRepositories(settingsDir, repositories);
```

Comment:
Are these actually remote repositories, or do they include the file:// local repository as well?

Suggest renaming to `asRemoteRepositories(...)` instead of `getRemoteRepositories(...)`.